http-client-core/src/main/java/io/micronaut/http/client/HttpClientConfiguration.java

```diff
@@ -161,6 +161,8 @@ public abstract class HttpClientConfiguration {
 
     private LogLevel logLevel;
 
+    private boolean allowBlockEventLoop = false;
```

Comment:
Use a a constant 

```java
public boolean DEFAULT_ALLOW_BLOCK_EVENT_LOOP = false;
private boolean allowBlockEventLoop = DEFAULT_ALLOW_BLOCK_EVENT_LOOP;
````
And use the constant in the javadoc of the setter with `Default Value {@value #DEFAULT_ALLOW_BLOCK_EVENT_LOOP}`. The javadoc of the setter should show in the configuration reference documentation.